Gemfile

```diff
@@ -15,9 +23,4 @@ group :test, :development do
   ]
 end
 
-gem 'idn', :platform => :mri_18
 gem 'idn-ruby', :platform => :mri_19
-
-platforms :rbx do
-  gem 'rubysl-openssl', '2.2.1'
-end
```

Comment:
This is no longer necessary. The Rubinius standard library is now packaged with the language, as it should be.